tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaListenerNameTest.java

```diff
@@ -44,6 +63,72 @@ protected void cleanup() throws Exception {
         // Clean up in the test method
     }
 
+    @Test(timeOut = 30000)
+    public void testMetadataRequestForMultiListeners() throws Exception {
+        final Map<Integer, InetSocketAddress> bindPortToAdvertisedAddress = new HashMap<>();
+        final int anotherKafkaPort = PortManager.nextFreePort();
+        bindPortToAdvertisedAddress.put(kafkaBrokerPort,
```

Comment:
I encountered an error `Listener 'PLAINTEXT://192.168.0.1/<unresolved>:15007' is invalid` when calling `super.internalSetup()`. Did you have the same issue when you run this test?